tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -2,9 +2,10 @@ jobs:
 - template: templates/win-ci.yml
   parameters:
     AgentPool : 'Win-GPU-CUDA10'
+    AgentDemands: 'Has19H1WinSDK'
```

Comment:
Note: this will cause the GPU build to run on only 2 agents - ncus-gpu-31wg, ncus-gpu-44wg because I was able to update the windows SDK only on these agents. Disk space on the other 2 agents (ncus-gpu-32wg, ncus-gpu-45wg) need to be cleaned up before the latest SDK can be updated on them.

Reply:
I fixed the other 2 agents as well. Now all 4 agents can be used.